### ISSUE TITLE
Making a mask for T2T-CHM13v2.0 ?

### ISSUE THREAD

**Issue by mpiersonsmela** [abyzovlab/CNVpytor#222]
We are using the T2T-CHM13v2.0 reference genome assembly, but we noticed that there is no option to use a mask for this (unlike hg38). Would it be possible to create a mask for this genome assembly?

**Comment by arpanda** [comment]
Yes, it's possible to add a mask file. Below is the command to create a mask file. The input mask file should be in the same format as the 1000 Genomes Project's [strict mask](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000_genomes_project/working/20160622_genome_mask_GRCh38/).

```
> cnvpytor -root MGSCv37_mask_file.pytor -mask ~/hg19/mouse.strict_mask.whole_genome.fasta.gz -make_mask_file

```
Once you have created the mask `pytor file`, it can be added to the reference configuration. 

For details please check: https://github.com/abyzovlab/CNVpytor/blob/master/examples/AddReferenceGenome.md

As of now, I'm not aware of any publicly available mask file for the T2T-CHM13v2.0 reference genome assembly in the 1000 Genomes Project mask file format. If you know, please share, I can add it with CNVpytor. 

Thank you,
Arijit

**Comment by mpiersonsmela** [comment]
Thanks, I'll give it a try!
